### DISCUSSION TITLE
Assembly Definition References UI

### DISCUSSION THREAD

**Discussion by sergiosolorzano** [lupidan/apple-signin-unity#93] [General]
after breaking the project testing settings I have moved this question to Issues https://github.com/lupidan/apple-signin-unity/issues/94

**Comment by lupidan** [comment]
This doesn't seem like an issue on our end.

I'll repeat it again here as well, you don't have to modify any files coming with the plugin, that includes assembly files and the import settings for native code and the main `.bundle` file for macOS.
